client/js/lounge.js

```diff
@@ -417,7 +417,7 @@ $(function() {
 		nicks = [];
 
 		for (i in data.users) {
-			nicks.push(data.users[i].name);
+			nicks.push(data.users[i].nick);
```

Comment:
I don't remember to be honest. But I added a TODO when I did the irc-fw conversion.